lib/ansible/modules/system/runit.py

```diff
@@ -265,6 +265,9 @@ def main():
     changed = False
     orig_state = sv.report()
 
+    if not sv.svc_cmd:
+        module.fail_json(msg="Could not find sv binary. Please ensure it is installed.")
```

Comment:
just add `required=True` to the `get_bin_path` calls

Reply:
So change it to `module.get_bin_path('sv', opt_dirs=self.extra_paths, required=True)`?

Reply:
i would do both svc and sv since the module cannot function w/o them

Reply:
nvmd, just realize they are the same command ... seems redundant